ui-core/src/styles/inputs/slider.css

```diff
@@ -0,0 +1,210 @@
+  width: 80%;
+  height: 10px;
+
+  .range-slider {
+    -webkit-appearance: none;
+    appearance: none;
+    height: 10px;
+    width: 100%;
+  
+    &::-webkit-slider-runnable-track {
+      -webkit-appearance: none;
+      appearance: none;
+      height: 4px;
+      border-radius: 2px;
+      box-shadow: 0px 0px 0px 0.5px theme('colors.black.25') inset;
+      background-color: theme('colors.black.5');
+      border-radius: 2px;
+      opacity: 1;
+    }
+    
+    &::-moz-range-track {
+      -webkit-appearance: none;
+      appearance: none;
+      height: 4px;
+      border-radius: 2px;
+      box-shadow: 0px 0px 0px 0.5px theme('colors.black.25') inset;
+      background-color: theme('colors.black.5');
+      border-radius: 2px;
+      opacity: 1;
+    }
+    
+    &::-webkit-slider-thumb {
+      -webkit-appearance: none;
+      appearance: none;
+      width: 10px;
+      height: 10px;
+      margin-top: -3px;
+      border-radius: 50%;
+      border: 0.5px solid theme('colors.grey.70');
+      box-shadow: 0px 1px 3px theme('colors.black.10');
+      background-color: theme('colors.white.100');
+      opacity: 1;
+    }
+    
+    &::-moz-range-thumb {
+      -webkit-appearance: none;
+      appearance: none;
+      width: 10px;
+      height: 10px;
+      margin-top: -3px;
+      border-radius: 50%;
+      border: 0.5px solid theme('colors.grey.70');
+      box-shadow: 0px 1px 3px theme('colors.black.10');
+      background-color: theme('colors.white.100');
+      opacity: 1;
+    }
+
+    &:focus {
+      outline: none;
+
+      &::-webkit-slider-runnable-track {
+        box-sizing: content-box;
+        border-radius: 4px;
+        background-color: theme('colors.white.75');
+        border: 2px solid theme('colors.primary.50');
+        box-shadow: 0px 0px 0px 0.5px theme('colors.black.25') inset;
+        opacity: 1;
+      }
+
+      &::-moz-range-track {
+        box-sizing: content-box;
+        border-radius: 4px;
+        background-color: theme('colors.white.75');
+        border: 2px solid theme('colors.primary.50');
+        box-shadow: 0px 0px 0px 0.5px theme('colors.black.25') inset;
+        opacity: 1;
+      }
+
+      &::-webkit-slider-thumb {
+        -webkit-appearance: none;
+        appearance: none;
+        width: 10px;
+        height: 10px;
+        margin-top: -3px;
+        border-radius: 50%;
+        background-color: theme('colors.white.100');
+        border: 1px solid theme('colors.grey.90');
+        box-shadow: 0px 1px 3px theme('colors.black.10');
+        opacity: 1;
+      }
+      
+      &::-moz-range-thumb {
+        -webkit-appearance: none;
+        appearance: none;
+        width: 10px;
+        height: 10px;
+        margin-top: -3px;
+        border-radius: 50%;
+        background-color: theme('colors.white.100');
+        border: 1px solid theme('colors.grey.90');
+        box-shadow: 0px 1px 3px theme('colors.black.10');
+        opacity: 1;
+      }
+    }
+
+    &:active {
```

Comment:
Even with the right size (see comment below), I can't make the `active` happen when holding the click on the slider 🤔 (we should see the thumb shrinked while we hold the click to change the value)

Reply:
Strange indeed

Reply:
actually it’s not the `&` it’s because active needs to be last for some reason :/

Reply:
Oh right got it